wrappers/wasm/demo_reader.html

```diff
@@ -12,6 +12,46 @@
 	zxing = instance; // this line is supposedly not required but with current emsdk it is :-/
 });
 
+/*
```

Comment:
This iterator seems a bit over the top, just to replace a `for( int ... )` with a `for( ... on ... )` loop in this simple demo, but I generally like abstractions and maybe someone learns something from it for a 'real' application. Keep it or leave it, as you prefer.